src/client/app/data.json

```diff
@@ -1,5 +1,18 @@
 {
     "masterConfig": {
+        "shopForm":[
```

Comment:
please remove the form from shopform label  
and also maintain proper naming convention